engine/src/main/java/org/destinationsol/game/screens/MainGameScreen.java

```diff
@@ -673,7 +675,7 @@ protected boolean shouldWarn(SolGame game) {
             float heroCap = HardnessCalc.getShipDmgCap(hero.getShip());
             List<SolObject> objs = game.getObjectManager().getObjects();
             FactionManager fm = game.getFactionMan();
-            SolCam cam = game.getCam();
+            SolCam cam = game.getContext().get(SolCam.class);
```

Comment:
and here

Reply:
This one is actually in a private static class, so it doesn't have access to the enclosing class's reference